tests/resources/test_dividend.py

```diff
@@ -0,0 +1,21 @@
+from fast_arrow import util
```

Comment:
do you need this import?

Reply:
I'll merge and probably take this out. Thanks for the PR.

Reply:
Thanks. The other tests had this and you're right its not needed. Yes, it can be cleaned up in a follow-up PR.